account_analytic_parent/views/account_analytic_account_view.xml

```diff
@@ -31,7 +31,7 @@
     </record>
 
     <record id="analytic.account_analytic_line_action" model="ir.actions.act_window">
-        <field name="domain">[('account_id', 'child_of', active_ids)]</field>
+        <field name="domain">[('account_id', 'child_of', active_id)]</field>
```

Comment:
Here `active_ids` is correct and I don't have any issue with it. Anyway to change it to `active_id` you should do:

```suggestion
        <field name="domain">[('account_id', 'child_of', [active_id])]</field>
```